2-collectors/scala-stream-collector/core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/metrics/StaticMetrics.scala

```diff
@@ -0,0 +1,21 @@
+package com.snowplowanalytics.snowplow.collectors.scalastream.metrics
+
+import java.io.File
+
+import scala.util.Try
+
+object StaticMetrics {
+
+  def javaVersion: String = classOf[Runtime].getPackage.getImplementationVersion
+
+  def scalaVersion: String = Try {
+    val props = new java.util.Properties
+    props.load(getClass.getResourceAsStream("/library.properties"))
+    props.getProperty("version.number")
+  }.getOrElse("")
```

Comment:
no need for all this reflection, you can leverage a sbt plugin we're already using to expose this build info: https://github.com/sbt/sbt-buildinfo

Reply:
No need for the reflection for both the java and scala versions